02LessonTwo/homework/index.html

```diff
@@ -6,5 +6,26 @@
u0Eh1IIGsbnmOB4jdxAMeiOAC4bB8o8MrL/3MSeslJ78vVgHkW50V9LNiLjcHLh2AzFSYj9WRRdPfp4EjjWZ0HKy4HCMN3zdU8Hi/dnzBBrMYpOymTLKcFF9AQ2gP/6OK74sMfS4/OtVdwrK2q1tpzL9bCRt/LKBWqKnPk1j9oKfKO3/AAwHN0HxP5U3bMuWD/Fh9zofyrvlB1WBfrTR/Bh+dV+k8JfCkLvzZv5QT+VLsD5ZoU7o7s2GeaeCRXEgYYiKWIlXjDrkkOcbu0o0N733byK2Bn6yNH+sqnzI199bHQnEqMbNGT2nhiZfBHcMP8wNYnaiymPeSL+JONwXaLnGQa3uAJ0UC5Nh2ZQADyNW1xkeKjzArLG4ta1wRxDKdx7jrRDl1vyv/vWTiEQHsADcDYWBtut3CudSs6tNHnPThR1mGwwJIiRnJNyQpOVB3kAHXfoKwf2Y1iVPZW9t+4Va6SY4vjsTY6LlS/IIutuXavTRj3yEdkAix9q+vnWsjkqo1hWNptjI9kkxq+cgNbhzPvq2vR87+sBHx99MTabhAqqlhuJBvYd1O/bMtrZE8e18L6VBvL0OuMcFbp8fJINlSFSvWkDUZdbW8f0rMhwbFL3sLlRvO7Sr6bSkYDsqNxHta1CkzhcnV5rFjcG283Pxpxc1z9hTx43TSfHz8GriklkUgKtjbc6ncb8PKottEmSOX+8T3lVb4q1VfnHG27uNPxsobDxm/sOw8BmuPRXPpXZF7nlSSp0yaJ7jfurJ6Tr+5B45x8DVrDPwFVtsjMIl5yIPW9XOc9DeARpFGNyIqjyFvyqKrW0fb8vzNVaiyh1dXV1ICPEDsmsropNeDJ/ds6eQY29xAq1trFdVGH+iCM/2ToT5XB8qw+iU+kzDUGZyO8G1aS2F1CuocZhVlRkcXVhY/qO8b6mBrqyMxOie0WhkOAnNyNYHP0lP0fjbvuOVF1CnSXZPXxgr2ZY+0jbjflfh+VaHRTbvzmMh9J4uzKN1+GcDv48j4itvfcE62Noi9CuycUdn7VU7ocV2W5CQnQ/zEeUhoroe6bbL67DsQO2nbXxHDzFx42oi6YTVo9ghkuARViRLjMOO/uoL+TrpB86wsbk3a2R+6VdCT9oWb7wo1wz8OdXOdFSSM71OVuf6jjQZ0YxnzTFYzByA5RnxkAAuera5lRR3NuHG5r0GTDg7tKBvlFgMHzfHqDnwcil7Xu2HkISUd+8eGtAwS2tiQ20oXsR1sE8NuGaMmTQ21FiutbdVenaKk2ClW2VcQY7gaZZ1OvnarVSyclcXAHdOzeXCLzkB9Cv61oTAdUAxAuba8b6WHrWX04BOKwgUgG72JBIvdd4uL+tWv2MHT947MSL2UBBcag9ntGxsdWO6s9EPqzM6LP8Auch3xu6H1v8AnWbPjHw+J+eRnMEm6phfgI108x1g+7V7Y5yYrEx8ysg89/4h6UvQXDRTYmfB4kfusWXVWG9cRESyFTwNmcd+a3GtJW2ZbpI9RGNZ0GujAG1l3HUXtvqrjMSI0LWLEblUXZm4Ko5n3bzpVKPYO0cIixokWNiQBUKuIpQo3Zlfsmw5GopNtSRm02BxkfeIc6/zKbVzvFJdDrWaDXJ5xtPYOITEqps808ckzLwLXdnRTxYBb+PiL0cPiRIMqg5uX6UYdIOkUDT4OeKTtwTAOrKysI2tmuGA4Aj71V+mewkhxrowtHOOuiYaFWb+0CnjZtbcjVtGqNvkisrxzpbowIMPMQbI7AfV599t1TQYOUBgUe7d351cwePkwYKyDrI2IIlW+/d2h/751ox7awzKLyZTa5IDam1t/GuLI8kXWnb4PSw+VNJ69+zMOMuAAY25btNK5JHOmRr79BfT1rWOPhJNpNNd19/dpp/tTcPiUDsQ4AIUct177/Gp6nv6Tr2VVP8Ao1JJn07XLgDu8qzZoC0TKRvUN59pW+KUkuEQW7IpmE/tGFzYRyWBJNvZOnoK7UqPD12zMwsm48wKXEteXDDnNH+IVDEdT9p/xGpT/wDJwn/ej/GtdBBnpO0Pb8h+dVas7Q9vyFVqiUOrq6upAVtoxB4yp1B0PgdD8aFeii9WZoTvR7+IOl/cPWjCf2T4UJwi20Ht9KIE95uv6VpcCYUYF79jj9H9P0/3qesyteb6J4lVJ8SNaQyOhfbWHfDzDFwe0ntrwePjfy0PkeFFFV8aNBQnQM09nY5J4kljPZcX71PFT3g1OwuLUG9A2y4nFxLpGBmC8AwYC45aG3pyozptUNO0DvQOT5rtGbBnRZx1kPIOtzYcrrnH3Fr1/Dy5lB48fGvEumDFMXgpE7LiVbMN+jJXtkYs7gbrg+ZGtWi7RCSpl2PEc/WmbX2euIglhb2ZUdCeWZSLjwveo6nwp3imI8a2qXl2Jc3E2GyBgTcrJh5MjXPPKCfOtmGYOquNzqrjwYAj3GleFc22I7djrpzl4XeBS3qdayuiLk4KAnXsW8gxA9wFYycG8XLRidM9MXgz3uPetb0egHdasXp2P3mDPHrbeRKVr4c9kVN8Ip1YL7STqtoRng6tH5gm3+io8DgeskxUQYo6vHNE43o+puPUVP06FnhYbw8dj4qpPvAp+D02ibcYNf5h+grXQyendDemC4gCDEWixiWDIbAS8ni4MDvsN2vCi2vItp4COWGTrFDGNSyHUFW39lhqNw0qX5J+kGJmjkWWZpBHlC5rEgd7bz53qkZWTkqPR9q7Iw+KUrPCko1HaAuPst7Smx3givNul3R8wr81kLTQuszYGQk9bh540L9UzfSjYCw8ByvRrtuVvmWMOYgiGVgQSCCEaxBGo3Chzbc7NgNjuxJZsTggWOpIaN1a543G+tGVwefbIx140Laq1lbzOUmsbC4gRrJEwuVLgaeQ996n2fpAe57DwzLTXgUz4m4vZ9PMteoTSp2dWG9SoXC45VAAF777gHXlY7q2jiI3AyoPHT0tw4UHYlBemw1KWCMlaOiHiXB01Z//2Q==" />
```

Comment:
I got it ! 

Thanks :D

On May 11, 2016, at 2:37 PM, Victoria Chuang <notifications@github.com> wrote:

> In 02LessonTwo/homework/index.html:
> 
> > +    Austin since I was five years old. I have been studying mechanical engineering but intend to study computer science but want to have the tools to have an extra income to help me through school.</p>
> > +    <h3><strong> Hobbies</strong> </h3>
> > +      <ol>
> > +            <li>Exercise</li>
> > +            <li>Qi Gong</li>
> > +            <li>Reading</li>
> > +     </ol>
> > +  <h3> Things I don't like </h3>
> > +    <ol>
> > +        <li> Cats AND FROGS </li>
> > +        <li> Mornings </li>
> > +        <li> Traffic </li>
> > +   </ol>
> > +  </h3>
> > +  <h3> TV Shows I like </h3>
> > +  <img src = "data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBxQTEhUUExQWFBUVGRQXFxgXFxYdGhcVGBgYHBcdFxcdHCggGBwlHBoUITEhJSkrLi4uFx8zODMsNygtLisBCgoKDg0OGhAQGywkHyQsLCwsLCwsLCwsLC8sLCwsLCwsLCwtLCwsLCwsLCwsLCwsLCwsLCwsLCwsLCwsLCwsLP/AABEIALsBDgMBIgACEQEDEQH/xAAcAAABBQEBAQAAAAAAAAAAAAAGAQIDBAUABwj/xABHEAACAQIDBAcDCQUGBgMBAAABAgMAEQQSIQUxQVEGEyJhcYGRMqGxBxRCUnKCssHRFSNikrMzU4OiwuEkQ2Nzw/A0RNIl/8QAGgEAAwEBAQEAAAAAAAAAAAAAAAEDAgQFBv/EACwRAAICAQMCBQMEAwAAAAAAAAABAhEDEiExQVEEEyIyYXHB4QUUobFCgZH/2gAMAwEAAhEDEQA/AAxjTiPZ+/8A6KcyU4HTwLfl+lc50ELNTTU5XSoXU0hjLafeH5j86ey0oWwPH2d/2hTmPdQxohrjUlqY9IBj6UrHVvtP+I0l6kcat9p/xGgCJhXMKdmqrtHHrGLnUncOZ/TvoSYNolZgBcm1uJ0qF8aCDlV3vbULZdGB9o2HA0Oy7Tdrk2vwP1R/CNwPfvqtJOze0zHxJNWjjXUlLK+hr4ra5B0QfzA/h3VANtN9Ue+suurWiJjXI38PtInUow7xqP8AarkEqtqpB8KGU7NmDWPde/6VebaKkarZuDj2vda/neh4k+BrK1ybj/Q/xP8Ax00resuPawsoN2tm4WOuXvIPs92+r2HxCvqpvbf3eVScWiqkmWKagtXClA1pDJBSyL7H+L/4qYKcfo/4n/jpAJbzpCKeBSM1MBA1OHfUdPAoAcx7J8U/EKcD32pJD2T4p+Na4EU
>  xHFqTzrg
>  KSgB4NOINNS1PuKYF0Uqj4/kaZpXNu+8PwvWQHkVHpTy1ML0jRGxsG1+r+Na4teul3H7n9RKWMC+6gRyUkiXp2WkcUhkGSklbtN9pviacTTHOp8aYhBWLjYAU6xrs8nsAfRT6PcNNb9/DfWripLIx5K3wrD2pizkjQaDq4ye/sgW91WwpbtksreyMqurq6tkySKMncM1tSO7w31M06ADLH2uJJJHkP1qujEG4JBG4jfRzsDoXisZFHMTAEe9mcNnsGK6gDXcbXNDkorcai5OkB+HhLdtiFUcT8FHGrcLxE6AseeRm9e0PcBR3iPkpkJH/ABKOO9GS3cACwt6Vo4P5L0C2lnYj6sahF8ySxY+NY/cY11N/t8j6AAFiYXZbWNs6EgqeGYNqPO4qGRwhzrlZrWOlswBsTp35fQ0Z4v5LXXMI8R2GtfMhLBR3hrH0FZ3TPo1HhYIJIgT1bZZWPtMGtYt3aEW/iFa8+Etu4eTOO/YylcMARuNj61IN1QYVbRoDvsKtxCoMqhgpX+j97/TTytMbh4t8BQgYt6QGuNOWgBBUitUZrgaBDnHZPin41pRSN7J8U/qJUopgNUUpWnClFADQtcTT6Q0AWiaVhoPtL+GSuNIw0+8PwvSAc4qE1OG0qFqQxrrdT938a1y6UpOh8vxLSM9MBTUZp5NMIrIzga5k1Pl+FaS9Kd5+5+BKAKm0V/dP9lvhQlPLmIPIKPQAUazm6MDxBHqKCFQm9gTbf3eNVxPZksvKG11dTkW5AGpOmn6VUkXMLh7I0jHgVUc2OlfQvR7A9RhYIjvSNFb7Vu1770C9CegrFkxGLUqFsYYD9G3stIOB45d99/KvQNp7QjgjaWZwiLvY33ncAACSe4VyeImpNRidvh8binKRapCwG82oIxPT55NMFg5pxraRwVTuI0Nx4larRfKUYmC4zBywbu0tyDz7LAaeBNR8qXYr5se56ACD30JdL8KGwmJQi9kcjxQZ
>  h71Fb2yt
>  oQ4uITRHMjaXsQQRvB7xVbaKKMwawS2tzpYjW5NEdmalujx/BvdVPMD4VPVTDqFZ4g4cRsQrAghkv2SCNN1XLV0Pk5FwOpkh3eJ+FOUU2fcPH8qEDGXp16bXUxEoWkIpA9dmpgOduyfFP6iVKDVeTcfGP+olTgUMEOC0tqWkNIZ1dakvT1FMCcWpCND4j4GutS5tD5fEUhDb1ETTm3UwbqAODCzeBpTUbbm+y3wqS2tAIeIxakkSmF7U0saQxNa5n7R+5/TSuduJIqFm7R+7+BaBj5W09aMOjOzJlwcRw+JMLyIpijKRtG7ZFdy4KM1szEFhu07gQ0UZ9BJVkgEUjHRZcIwvYgEs8eVhquZGcX4mNeIrMtkahuwU6Y7BDxLilSONw7Q4tYiDGso9mRQPZV1sTyzDjcmn0HlSDE9eYXmWJWuYrNkZ9FY3soFg/Hea9I2DiYo8Y+FiKRRpGHWMQvG7tmYEF3N5VA7VwNc2hsCCSTThGjG7rGKA/wAWVmHrlI8bDjQ8tLTQLCm9SZiYPpvgpLXm6onhKrJ/mPZPkau4/CxYgLKqpOVFkIyuFvrdQWCAni1wdAL1JtDYOHmv1kSkneR2SfEi2bzvQV0k6CRQRvPEbqgvIhFi0Vx1gBQqNFudVvpvqcVBvZ0Vk5pbpMt4hZl7cksjKwYoIReJRePq9YlMxOQy3uAMwUXI1rXjwKO8KBpZY3htMksbsomABJVpEuQbkHUqCNL37O7D0Q2eVDR4WAqQCCFBuCLg3O+440jdDsAf/qQfyCquS01X8fkioPVqv/V/gzNsYdcLhD1KCNI2RiFFgFzjMzAakLfOQN4W24155NgYpnZ8XNI4t1p6zERIdSoskFtG7Q7AN9COFqNn6PL86lXCSPhoURY5VSzI8pJJUK9wpVMl7D6YGhBrK6OdFlkwmISRneWJ5o1QyOEQRgBMsYIGos1ze4YHmaMdKwyW6o88wsUYeVor9WWITMRcq
>  DvO7fVoP
>  eiODYMBFzEt7XObWwA1JJ3DjehbsXdoxlRmJUD6ugGh1FwL+dUkicX0LV6SXcPH8jUJanO3ZH2h+FqykabHE0lMBpb0CJBSg0xaeBTASX2fOP8AqJVmq857PnH/AFEqYtSGOalWmF6jLUAWiaUMKrq2lcDQBeNNBFm+7+JaaHrg2h+7+NaBD9KaxpBXMKBkTr7X2X/CaUsa5jo32X/AaaxtQIa4pueuY6VEwoGOkNIDr5L8BTGBFSqnHuFACgVNgcX1EhYp1sMoCzxA2YhTdHjPCRDqNeHDeIC9KX0vwpDD/ZG1MBLJhwsr4rEKXETSK3WxqQS3WEgCygEXNzy50T7SwKTxmN72NjdTZlZSCrKw9lgQCD3V5/8AJg2WLaGJAvYi3f1aSNYHvuPdXpV+VQyKpbHRjeqO5k7OxEySGDEMJLgtDNlymQD2kkA0Ei6G4tmFzYZTWqygggi4OhB3Ecb0jxg2uL2Nx3HUXHkSPM06pt2bSoxsBsebDgphsXJHFrljdI5FQHghYZgvIEm1Mxb4nroo5MbIElEgJjjgjJkAUqoYoSCV606G/YrbB1tXEX761rZny4mfsWKJVKwPnS9hY5gCCc/b+kxfMWO8k67qD+mOLEM80uHmkiZIbzmJhYz3VcOHBBGe2a+l7AbqONpwK8bIwuGFiOY3+mleY/KBZBBho0VI2JkOQAC6XFiALfSBreLeVmMu0QaTEysgV5ZGFtVLHLvvu8b02uNJeui7OdKhb0rNoPtD8LVGTStuH2h+FqAJL0oNUircmJvvBG7gAu7dzpyznedQN5AIK/aX9KB0XQ1cHqMGnUCHSt2fOP8AqJVg+NU5fZPin40qZXoAlA130hqMmmk0ASZqVWqK9KDTA0cp/wDbU8qQP5fxLVBsZIBqi2+1XfOpCBog3ce8Hn3VlpgmnwaWSo7VittyQEjKNCRuPDzpjbdf6o9D+tGlj1I2Jk7LfZb4GnWrDfbbkEWXUE
>  eyeP3qb+
>  2JOAH8p/WjSxakbLJXMp76xf2vJyH8p/WmttiTu9P96NDHrRtpHrU1/gPzocba0h5egpv7Sl5+4UaGGtdggMfM24+FY21oGeA4i/7sTdSg5kJmZvw+tSbFw82Ml6rMcg7UhAGij8zuA/SjTY3R84zYWWEXlimeUJxcgsrJ4lDp3gVSEK5JTneyCP5N9k9Vs5Fcaz5pHHc4sv8AkC+tXuiu0cynCym2JwwyMp0Msa6JKg+krLlvyPlQR0N6bnDxrHicz4cdlJgCWit9CVRrYcDv5XG4+xuzcNjUSS4a2sc0TWdPsSLqPDd3VyTTTerqdseFXQ1q6hiZ8fhdezjoRzsk6jxHYksL7wCav7H6SQYg5FYpKN8UoySD7p9rxW9TcXyjakuGawXf37/S1UsXglC9iInUkrFI0RYneQFZVYn+Ijxq/UeIgDqVa9jvysyn1Ug0kzTRgFIwbdRjI77wJ2UHvNsRv799APTfGiTFhBugSx1J7b2JGY77ALrzo92vhocKrTKCuRHYjM1jYHQi9jewtevC5cRIzM5Y3dizWJ1Zjc104o3uc2V1sbdqQpWFnf67eppMzX9o+pquj5I6vg3bUkh9n7X+lqxSG+sfU0hVvrH1NGn5C32N4VHKcrK4F7EBhzU8/wBawyDfeaTKedPT8gpPojfMXVuU+iQHjP8AA3Dy5eNS3qPbEAbCQyB0dl1bKSSiyFiFbT2gyyk9xWh4rSSs1k9L2CKc9k68U/GtLnHMetDeWuArWgnq+AlDr9YeopM6/WHqKHAtcVo0hqfYIeuX6y+opfnCfWX1FDlqUCjSGphL1em41aVrDRGY9yk28a12nOnaG820toNKsbLxJBuX0sdCd54X5aXFSlK0VxxcWeeTydo6cT8TUTSVcx6/vXNx7TnT7RqnKKqTYokpwlFtxphIqQOLa0mhpjeuHI015LjQU+Ug7qiPIak6AChIG6GFqswYGSRHkUdhBdmOg8BzPdR
>  HsLoeWs+
>  J0HCMbz9ojd4DXwrX6WrbDpBGAvWyRxqBoAL34d4HrTtXSMW6NHoJswRYVWt25u2x/hPsD018WNa3yb4z5visVhW0DN16fZewe32SF99W0iCAKu5QFHgNBQ90oR4WjxkI7cBuw+vGfbU91viaUZeoc4+k3envQl8z43Apmc3OIw4Gko4vGvF+a8d41JBBdkzAXlwczYd/poNUv/1Ij8R5Cvaejm2UnijdGujgFD8VPIg3HiCKz+lHQPDYxjKM2GxP99F9I/8AVj3P3kWJ51qcL4N4PEKK0zVr+V9GC2x+nWVljxyCItos6XML+J+gfHztW/tnYMOIXtorjeOY70car5GvPukOw8ZgFb5zEMRhjoZYxdGF9M66GNu/TXceNQdGuk+LiQph1WSANeMT5iypxVSp0F9191c0sbW62OtOM9oepduq+oWR4LF4Zi2HnMqm14cTc7vqyjtKbaajle9M2j05mw6qZ8EULNlW00bBm7ra28uNZ2L+UJnYKuEbMoGYGRRYnfl07Q7/AIUK9Itq/OZo5FimIivnDWvmOoCqDYKNO88aahfuRh2vbf8Az8F7EbWfFMmHxR+bQzyyM7hs+pYsiXOiKDl7tL6CsLpP0XnwL/vBnjY2SRQcp7j9Vu4+V6ubTnSXDFkPaTKSD7QbMN49RXsfRyKDG4JdA0bqOyw0A3MhHDKwYC26wtuFXx7rgh4uPlzWl2mj50D05b8q3+mPR+PDys2GlSaHNY5XVjExv2XI3jQ2busdd+FIjRsVdSp5EW8/CtNElKxuY8qXrTyFOVuIHpTQh5Vk2MZuNdanZb8DV3YuDEuIhhclVlkjQsN4DMAbd9OzO4e/JR0Uhx0MyzSuBfqyiZRYdl0Ykgk3OcW00B8h/ov0CnxmNmwmZYmw+frXZSQMj5AAOJJOm7QGvROi+y02btERxluqxUZKhjciWA3IvbcY2kI86OdlbORMbjplFjMuEDd7Kslz3XGS/wBm
>  9EWmrQp6
>  k6Z4jtf5MZY3aOPEwSOt7o4khYn+EyLkIPA5gDQhtDZM2HkMU0bRSD6Lgg24EcGU8CLg171t7pJizimgw8GHxCwxiVxPcMc7MFSJh7JsL3OmvC2of8qPSuHEYWHDnCtBiY5FYI6nNHHZs2R9zIxsN+ttwsKNXQEnyzy7qDXfNzYnlU4duC++uu9tV99Y1MtpiVMlOVKmSNuA3a0jKRvGtOzOlHpkuzY8pIzjS9i+l7crXqvhsBGwGZb5hv8AH86tZWUFetU3/gcm3Go1ZgCBLoLi+TmTzO+ot7GoxuXAAYrCLmIHAsNe4nlUTYUcKfjnIZrE6M2tt+pqJGJ+kfWq7mFXFD3wg5cKgKAC5FTRxSSyCKMl2PoBxueA76J4Og6lR1srFuOW2UeFxc+6j6sLS4QObM2XJiDaJbL9JzuHnz7hrR1sPo9FhxcDPJxc7/uj6I9/fVnZ+HeJQhKug0BChSB3qND4i3gd9XqTYvlnVh7WXrMbgYv+oZCPsWI/C1blDmLxbQY8Yh4JJIkjyKUANiRqbebDW1EeRMN231HNGGBB3GsXB9L8HJul6s8pFK2+9qvvqTZm1DiZWMf9hEbZv72TkP4FGveSKKZu0U+h2OOCxbYKQ2hmOaAk+y5+jfv3eIH1q9bwGJzdk7+fMd9eUdM9kGaHOmksXbQjfpvAPp5gUXfJ90iGKgjlPtj93KOTgDXwOh8zyq0HaOecdLH9F+kbY6Wd85ECSmFIx7JQAXeTi5e+46AC1t5PnGKjGGx2JiEZgAkzLHyVgCTGeKX1W3C27dRf0M2euHmxkI7LRTMrD60bEvA/ccrML8QB9UVv9I9gJtGAoQBiIwWw8o0ZXG5WPFGOhHfffROOqNFPD5/KyajzXb0AeE4gaSQ63H0l7/GomwtyJo95AzC2hB1Fx8Dw86v7HHX4Mi1mawYHgVIDAjxBqQDq2sLdnTuItu8K4m62PpIxUnqXDpg9t7Z/7v5wqhWQi
>  /I21see6
>  9enbKwUOC2MkeJBYSoFZBctJJNciNANSbtbyvuoK6SyCTDmNBlJ9Af96LsY6bTwwlhYCWFY8iNvw+JRs3aHAMVVc31QbbzXTgdo8j9Thpmmlt9zP6SwmLYpSdRnEUMVtP7UlbBbaXFi2n1TQjsTErInzbExjPEApVxrYaZlPpu8a29tbWfaE0SwgBMKiu0cgYA4tr5lY/wAWB4Ed9D+zY+rmc4k5cTIdzCwy8BG25hu3HgBWpuzggqJ8Z0JibWKRojyPaX/APQ99De0tnSYZgs6ixvlddVbn59xse6vSoksAKhx+BSZCkgzKdfA8CDwOpqeruUW3B5awuSRfhbQ0kU5jZZB7SMrDxUgitDFYVsNMYnPZ3o3NeH/ALzFOVesyxrYtIyIBodWYCi6dG1HUrs9h6cgrEuIS2bDOsy678puy+DJnHnRZsjbWHZBL18YEojK5nVSUsSuhI1s1j4V5ttWNMTtDE4fGYqSCNBF1EQkEaSoy9u5Is+oHfqeVaMHRvZMZEYjhZzuDSF3PkWJ9KnCehUzWTG8krQ7pNMiSfPcPiYRJGhWRS6sksQNwrKpLA33Fdbm1jesjpcVx2zuvKCOaEdYBfVbG0sbEgG9tbW3qK3cR0V2cyMPm8YuDuDKwPCxBvXme2ttyxxSYAknJI6mRj2mgFurHeT8LCmmpu1yg0uCp9f7MSHEKDc3G62lWTi05H0NUARzqUMOdNpFIt90WVxI0spy8dD7qhxfbIyg6c9KuQSKIwLgG3P8qgzLfePUVhOmbcdUd2GmPnYqzEBAhsc2pOgb2QbWsRrfyrNhu5uxJ7hoB3WFvfVyRM8cyKQT2W56srKB/lX1qjs5rqG5gH1qsErZyTbpGlFhoR/y19BTMbFGqO2RbKrN7I4Amlgqt0jmy4aXvAX+YgH41UkTdCsCqYZXtZ5LljxIDEKPCwB8636q7LhyQRLyjjv45Rf33q1UXyUXB1dXV1IDqjxE6opZyF
>  Ubyd1SVD
>  jMKsqNG4urCx/Ud430ACwwrbSkPVqIsMh7czKLm3Be/uHnvtR1gMIkUaxxjKiiwHHxPeTqfGhvodi2TPgJT24yXhP119oge8+Z+rRWpvW5dgj3OIoS2bP+ztpa6YfF9luSvfQ+TH0c8qLqxelmyfnEDKPaGq/aG711HnRF0wnG0FWBTNtiS1iPmcBl/wC6JGCX78nutRSseWVWGgOh8d4oA+RedZIpnZmbEFrTl/a0AEfflCggd4avRmW9XOc8D2BipIHeOUZS7SOl9zgOytY8w6t760BKGJ1ueNekba6GQYyJ1vlYSPJG6jtwSvrJYcVZu0VO/MT9Ujyza+zsRgZAmJWwJtHMv9lJ5/Rbmp/SubLi3tHteC8atKxz6GnEI2Ur9Px31X6HYCR9ov1LlHWBjceyxzgKsi/TUjNcb9LgggEVsW5CiRd41/O1bHQrENFh8TjIxmmxMi4fCgje4B7X2VJkY34R0sC3s3+pzXl6erZDs7EL+0McFtlzxk24OVIk9HDelbeOwUcylJFDqeBG7w5Gh3Z+yhhsXiIgxayYYsxvd3ZSXY35sWNFINbn7jycftAraMUmAaPJIXgd1TI+uQG/stvtYHTu40QSnsnwNZPyjr/wwPJ0I94/Orry2iJPML/MwA+NJj4Zi9McJ1uH6wDtwm/3T7X5HyrK6KDBy4kDGWWIx9m5KL1pIPbcEW7Ia1zbSid1zKyadsFdd2vPu4HuJrG+TvYzytiohOIHXIDG0ccgcAtfOj6MoNt31vCh1pscbuu4YYvoBgpwOrmlCjcEmDqPANmt5GqL/JjlFop0I5S4eNv847Q8qhxXQ3FRHPEkLMNzYZ2w8m+5BUgxkbtLV2z9vYuCdYcWDGjDSSZcnaJ0USR3RtONhcjhUk5f4ystUb9UaH9H3ODk+a4oMsjEiOXrHaKXXRVzGyMN1iATYc9Rnbc6Pjp5FsVDIhPNkUK1r8Ljz30bdODH8zl6zebdXz6
>  6/Yy99/d
>  evMYYVKgka2FzzPOhu1fcpBNSSXTctwRgu+76PKpnwim17aVQiRd1rnwOlJJENbC3dWGt+Sq43iWZcAOB+FVJsHqNRu18av4rAKsKsq9oqpv7zpVCN7KKcZNrZk8kUnTQd9YTMNAoZHAsALutnH4TWVghvA4M4A7sxy/5StXpI1jMcnWSMVdPb3WZgra3+qTWfOuSeRe8H4r/AKR61eL3ORr0l6MWNZPSxz1OX6zIPz/KrrNres3bZzdSDuM0f51UkHM62YjlYegplKs4kUSDc7SFe9A2VT4EDN96kqLKHV1dXUgGs4Fr8TYeNifyNOqnthT1LFfaWzL9pSCvvAqxh5g6K67mAYeBF6YGT0k2ezBZ4dJ4DmUjewGpXv528Rxog2LtJcTCsyaZvbX6kg9ofmPGoaH+u/Z+J6zX5riDaUD/AJb8GA9T6jlWlvsHG4Z0hFL4G4NiCNxB3EV1ZNgrs/HfszaiynTD4rsS8gSR2uWhytfkXr2uvJOleyvnEDKPaGq/aG711H3qKPkr6RfOsGqOf3sH7t77yAOyT5e8GrQdo55qmFWIwd2Do7RSCwzrbUA3yupBV137xcXNit71WxydbG0OMijkicWZ0ByHvkiJLR67mVmtvJW160wK6tmTw3pVsOTZsvVMS+Gku0Eh1IIGsbnmOB4jdxAMeiOAC4bB8o8MrL/3MSeslJ78vVgHkW50V9LNiLjcHLh2AzFSYj9WRRdPfp4EjjWZ0HKy4HCMN3zdU8Hi/dnzBBrMYpOymTLKcFF9AQ2gP/6OK74sMfS4/OtVdwrK2q1tpzL9bCRt/LKBWqKnPk1j9oKfKO3/AAwHN0HxP5U3bMuWD/Fh9zofyrvlB1WBfrTR/Bh+dV+k8JfCkLvzZv5QT+VLsD5ZoU7o7s2GeaeCRXEgYYiKWIlXjDrkkOcbu0o0N733byK2Bn6yNH+sqnzI199bHQnEqMbNGT2nhiZfBHcMP8wNYnaiymPe
>  SL+JONwX
>  aLnGQa3uAJ0UC5Nh2ZQADyNW1xkeKjzArLG4ta1wRxDKdx7jrRDl1vyv/vWTiEQHsADcDYWBtut3CudSs6tNHnPThR1mGwwJIiRnJNyQpOVB3kAHXfoKwf2Y1iVPZW9t+4Va6SY4vjsTY6LlS/IIutuXavTRj3yEdkAix9q+vnWsjkqo1hWNptjI9kkxq+cgNbhzPvq2vR87+sBHx99MTabhAqqlhuJBvYd1O/bMtrZE8e18L6VBvL0OuMcFbp8fJINlSFSvWkDUZdbW8f0rMhwbFL3sLlRvO7Sr6bSkYDsqNxHta1CkzhcnV5rFjcG283Pxpxc1z9hTx43TSfHz8GriklkUgKtjbc6ncb8PKottEmSOX+8T3lVb4q1VfnHG27uNPxsobDxm/sOw8BmuPRXPpXZF7nlSSp0yaJ7jfurJ6Tr+5B45x8DVrDPwFVtsjMIl5yIPW9XOc9DeARpFGNyIqjyFvyqKrW0fb8vzNVaiyh1dXV1ICPEDsmsropNeDJ/ds6eQY29xAq1trFdVGH+iCM/2ToT5XB8qw+iU+kzDUGZyO8G1aS2F1CuocZhVlRkcXVhY/qO8b6mBrqyMxOie0WhkOAnNyNYHP0lP0fjbvuOVF1CnSXZPXxgr2ZY+0jbjflfh+VaHRTbvzmMh9J4uzKN1+GcDv48j4itvfcE62Noi9CuycUdn7VU7ocV2W5CQnQ/zEeUhoroe6bbL67DsQO2nbXxHDzFx42oi6YTVo9ghkuARViRLjMOO/uoL+TrpB86wsbk3a2R+6VdCT9oWb7wo1wz8OdXOdFSSM71OVuf6jjQZ0YxnzTFYzByA5RnxkAAuera5lRR3NuHG5r0GTDg7tKBvlFgMHzfHqDnwcil7Xu2HkISUd+8eGtAwS2tiQ20oXsR1sE8NuGaMmTQ21FiutbdVenaKk2ClW2VcQY7gaZZ1OvnarVSyclcXAHdOzeXCLzkB9Cv61oTAdUAxAuba8
>  b6WHrWX0
>  4BOKwgUgG72JBIvdd4uL+tWv2MHT947MSL2UBBcag9ntGxsdWO6s9EPqzM6LP8Auch3xu6H1v8AnWbPjHw+J+eRnMEm6phfgI108x1g+7V7Y5yYrEx8ysg89/4h6UvQXDRTYmfB4kfusWXVWG9cRESyFTwNmcd+a3GtJW2ZbpI9RGNZ0GujAG1l3HUXtvqrjMSI0LWLEblUXZm4Ko5n3bzpVKPYO0cIixokWNiQBUKuIpQo3Zlfsmw5GopNtSRm02BxkfeIc6/zKbVzvFJdDrWaDXJ5xtPYOITEqps808ckzLwLXdnRTxYBb+PiL0cPiRIMqg5uX6UYdIOkUDT4OeKTtwTAOrKysI2tmuGA4Aj71V+mewkhxrowtHOOuiYaFWb+0CnjZtbcjVtGqNvkisrxzpbowIMPMQbI7AfV599t1TQYOUBgUe7d351cwePkwYKyDrI2IIlW+/d2h/751ox7awzKLyZTa5IDam1t/GuLI8kXWnb4PSw+VNJ69+zMOMuAAY25btNK5JHOmRr79BfT1rWOPhJNpNNd19/dpp/tTcPiUDsQ4AIUct177/Gp6nv6Tr2VVP8Ao1JJn07XLgDu8qzZoC0TKRvUN59pW+KUkuEQW7IpmE/tGFzYRyWBJNvZOnoK7UqPD12zMwsm48wKXEteXDDnNH+IVDEdT9p/xGpT/wDJwn/ej/GtdBBnpO0Pb8h+dVas7Q9vyFVqiUOrq6upAVtoxB4yp1B0PgdD8aFeii9WZoTvR7+IOl/cPWjCf2T4UJwi20Ht9KIE95uv6VpcCYUYF79jj9H9P0/3qesyteb6J4lVJ8SNaQyOhfbWHfDzDFwe0ntrwePjfy0PkeFFFV8aNBQnQM09nY5J4kljPZcX71PFT3g1OwuLUG9A2y4nFxLpGBmC8AwYC45aG3pyozptUNO0DvQOT5rtGbBnRZx1kPIOtzYcrrnH3Fr1/Dy5lB48fGvEumDFMXgpE7LiVb
>  MN+jJXtk
>  Ys7gbrg+ZGtWi7RCSpl2PEc/WmbX2euIglhb2ZUdCeWZSLjwveo6nwp3imI8a2qXl2Jc3E2GyBgTcrJh5MjXPPKCfOtmGYOquNzqrjwYAj3GleFc22I7djrpzl4XeBS3qdayuiLk4KAnXsW8gxA9wFYycG8XLRidM9MXgz3uPetb0egHdasXp2P3mDPHrbeRKVr4c9kVN8Ip1YL7STqtoRng6tH5gm3+io8DgeskxUQYo6vHNE43o+puPUVP06FnhYbw8dj4qpPvAp+D02ibcYNf5h+grXQyendDemC4gCDEWixiWDIbAS8ni4MDvsN2vCi2vItp4COWGTrFDGNSyHUFW39lhqNw0qX5J+kGJmjkWWZpBHlC5rEgd7bz53qkZWTkqPR9q7Iw+KUrPCko1HaAuPst7Smx3givNul3R8wr81kLTQuszYGQk9bh540L9UzfSjYCw8ByvRrtuVvmWMOYgiGVgQSCCEaxBGo3Chzbc7NgNjuxJZsTggWOpIaN1a543G+tGVwefbIx140Laq1lbzOUmsbC4gRrJEwuVLgaeQ996n2fpAe57DwzLTXgUz4m4vZ9PMteoTSp2dWG9SoXC45VAAF777gHXlY7q2jiI3AyoPHT0tw4UHYlBemw1KWCMlaOiHiXB01Z//2Q==" />
> Ah! So in this case, your HTML would be looking for a folder called "users" inside its current folder, which is the homework folder! Step out of the current folder using "../", so it would be img src="../../archerlogo.jpg"
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>